src/main/java/hudson/plugins/throttleconcurrents/ThrottleQueueTaskDispatcher.java

```diff
@@ -178,6 +188,106 @@ else if (tjp.getThrottleOption().equals("category")) {
         return null;
     }
 
+    private boolean isAnotherBuildWithSameParametersRunningOnAnyNode(Queue.Item item) {
+        if (isAnotherBuildWithSameParametersRunningOnNode(Hudson.getInstance(), item)) {
+            return true;
+        }
+
+        for (Node node : Hudson.getInstance().getNodes()) {
+            if (isAnotherBuildWithSameParametersRunningOnNode(node, item)) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    private boolean isAnotherBuildWithSameParametersRunningOnNode(Node node, Queue.Item item) {
+        ThrottleJobProperty tjp = getThrottleJobProperty(item.task);
+        Computer computer = node.toComputer();
+        List<String> paramsToCompare = tjp.getParamsToCompare();
```

Comment:
In corner cases there may be an NPE here, because ThrottleJobProperty may be deleted during the run.
I'll fix it in https://github.com/jenkinsci/throttle-concurrent-builds-plugin/pull/42